card/src/main/java/com/adyen/checkout/card/CVCVisibility.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by ozgur on 5/9/2023.
+ */
+
+package com.adyen.checkout.card
+
+enum class CVCVisibility {
+    SHOW_FIRST, HIDE_FIRST, ALWAYS_HIDE
```

Comment:
Did you change the functionality of `SHOW_FIRST` to `ALWAYS_SHOW` already?

Reply:
The functionality was already the same
```kotlin
            CVCVisibility.SHOW_FIRST -> {
                when {
                    cvcPolicy?.isRequired() == false -> InputFieldUIState.OPTIONAL
                    else -> InputFieldUIState.REQUIRED
                }
            }
```

Reply:
Shall we rename the field then?

Reply:
done